py3status/modules/whatismyip.py

```diff
@@ -13,14 +13,8 @@
         (default 'ip')
     negative_cache_timeout: how often to check again when offline (default 2)
     timeout: how long before deciding we're offline (default 5)
-    url: change IP check url (must output a plain text IP address)
-        (default 'http://ultrabug.fr/py3status/whatismyip')
     url_geo: IP to check for geo location (must output json)
-        (default 'http://ip-api.com/json')
-
-Format placeholders:
-    {country} display the country
-    {ip} display current ip address
```

Comment:
> TLDR: in the default configuration {country} changes to {country_name}

can't we be clever and if the url is `freegeoip.net/json` then just create country in the dict as well as country_name then the placeholder will still work.

We could also do query -> ip for `ip-api.com/json` too so that remains supported

And yes I think we should keep the placeholders but you can add ... if other items are returned then auto placeholders ...